matminer/featurizers/structure.py

```diff
@@ -952,24 +956,30 @@ def featurize(self, s):
     def feature_labels(self):
         if self.stats:
             labels = []
-            for attr in self._labels:
-
+            for attr in self._site_labels:
                 for stat in self.stats:
                     labels.append('%s %s' % (stat, attr))
             return labels
         else:
-            return self._labels
+            return self._site_labels
 
     def citations(self):
-        return ['@article{zimmermann_jain_2017, title={Applications of order'
-                ' parameter feature vectors}, journal={in progress}, author={'
-                'Zimmermann, N. E. R. and Jain, A.}, year={2017}}']
+        return self.site_featurizer.citations()
 
     def implementors(self):
-        return ['Nils E. R. Zimmermann', 'Alireza Faghaninia', 'Anubhav Jain']
+        return ['Nils E. R. Zimmermann', 'Alireza Faghaninia',
+                'Anubhav Jain', 'Logan Ward']
 
+    # TODO: Should we make these classmethods? They call should return an initialized version of the class -LW
```

Comment:
I don't have a strong opinion, in my experience:

1. the class method adds the complication of remembering to put "cls" as the first argument
2. the above is useful if you need the class information for something (e.g., you write a subclass that inherits from this class, and want the same function to work for both the superclass and subclass but initializing a class of the same type).

I don't really see a case for point (2) above here, let me know if there are other reasons to prefer a class method

Reply:
Thanks for your thoughts on this. I agree, case (2) is the main advantage of a class method and we haven't yet seen a need for that.

I'll remove this Todo